File: Java/Code Related to Arrays/54. Find Largest element in an array.java
import java.util.Scanner;

class Main {
    public static void main(String [] args) {
        Scanner userInput = new Scanner(System.in);

        System.out.print("Enter the number of elements of the Array: ");
        int size = userInput.nextInt();
        int[] numbers = new int[size];

        for (int i = 0; i < size; i++) {
            System.out.print("Element " + (i + 1) + ": ");
            numbers[i] = userInput.nextInt();
        }

        int largest = numbers[0];

        for (int i = 1; i < size; i++) {
            if (numbers[i] > largest) {
                largest = numbers[i];
            }
        }
        // Method 2

        // for (int num : numbers) {
        //     largest = Math.max(largest,num);
        // }
        System.out.println("The largest element in the array is: " + largest);
        userInput.close();
    }
}

File: Python/Code Related to Arrays/54. Find Largest element in an array.py
lengthOfArray = int(input("Enter the length of an Array:"))
array = []

for i in range(lengthOfArray):
    element = int(input(f"Enter {i+1} element :"))
    array.append(element)

if array:
    largest = array[0]

    for element in array:
        if element > largest:
            largest = element
    print("The Largest element is",largest)

else:
    print("No elements are entered")

# Method 2

# lengthOfArray = int(input("Enter the length of an Array:"))
# array = []

# for i in range(lengthOfArray):
#     element = int(input(f"Enter {i+1} element :"))
#     array.append(element)
# array.sort()
# print (array[-1])

# Method 3
# print(max(array))